pkgs/spectest/src/containers/state.zig

```diff
@@ -4,6 +4,8 @@ const configs = @import("@zeam/configs");
 const types = @import("@zeam/types");
 const ssz = @import("ssz");
 const params = @import("@zeam/params");
+const stf = @import("@zeam/state-transition");
```

Comment:
Can we follow the leanSpec style to migrate util function into `BeamState` member function like previous?